src/common/utils/index.js

```diff
@@ -24,6 +24,33 @@ export function sortByForgeVersionDesc(a, b) {
   return 0;
 }
 
+export const formatNumber = number => {
+  // Alter numbers larger than 1k
+  if (number >= 1e3) {
```

Comment:
Codacy found an issue: [The numeric literal '1e3' will have at different value at runtime.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)